tests/components/ads/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for the ADS component."""
```

Comment:
Ideally, there should be tests for `__init__.py` as code coverage is lacking

Reply:
added tests.